modules/logging/efk/README.md

```diff
@@ -1,3 +1,35 @@
-This chart installs an elasticsearch cluster with kibana, elastic-curator and filebeat by default. You can optionally enable logstash and install Fluentd if you prefer. It also optionally installs nginx-ldapauth-proxy.
+## Elastic Kibana Filebeat
 
-[Elastick-stack Documentation](https://github.com/helm/charts/tree/master/stable/elastic-stack)
+https://helm.elastic.co/
+
+
+## Prerequisite
+
+Create Google OAuth keys
+First, you need to create a Google OAuth Client:
+
+* Go to https://console.developers.google.com/apis/credentials.
+* Click Create Credentials, then click OAuth Client ID in the drop-down menu
+* Enter the following:
+```
+    Application Type: Web Application
+    Name: Grafana
+    Authorized JavaScript Origins: https://grafana.mycompany.com
+    Authorized Redirect URLs: https://grafana.mycompany.com/login/google
+    Replace https://grafana.mycompany.com with the URL of your Grafana instance.
+```
+* Click Create
+* Copy the Client ID and Client Secret from the ‘OAuth Client’ modal
+
+
+## Example how add with module
+```
+module "efk" {
+  module_depends_on = [module.argocd.state.path]
+  source            = "../modules/logging/efk"
+  cluster_name      = module.kubernetes.cluster_name
+  argocd            = module.argocd.state
+  efk_oauth2_domain = "auth"
+  domains           = local.domain
+}
+```
```

Comment:
can you please add a table of input parameters and outputs, you can simply generate it by `terraform-docs markdown .` and past it here as is